frontend/lib/src/components/auth/components/logout_dropdown.dart

```diff
@@ -0,0 +1,65 @@
+import 'package:dropdown_button2/dropdown_button2.dart';
+import 'package:flutter/material.dart';
+
+/*
+  Reusable logout dropdown component.
+
+  @param defaultValue, the default value of the dropdown.
+
+  @param selectedValue, the selected value.
+
+  @param onChanged, called when the user selects an item.
+
+
+*/
+
+class LogoutDropdown extends StatelessWidget {
+  const LogoutDropdown({
+    super.key,
+    required this.defaultValue,
+    required this.selectedValue,
+    this.onChanged,
+  });
+
+  final String defaultValue;
+  final String selectedValue;
```

Comment:
```suggestion
  final String x;
```
change variable name since there is no changing of selected value